### DISCUSSION TITLE
Resolved: Milestone usage

### DISCUSSION THREAD

**Discussion by tcharding** [rust-bitcoin/rust-bitcoin#1278] [General]
IIUC we were going to use milestones to track PRs merged to assist in writing changelog notes. Well I just spent 5 minutes doing mindless clicking and thought "there must be a better way than this"

Query: `is:pr is:closed merged:>2022-08-31`

Returns all PRs merged this month, obviating the need for milestones as described above. Is there any other use for milestones?

It seems this cannot be done via the Rest API so `gh` users are out of luck.

**Comment by Kixunil** [comment]
Some open issues/PRs don't need to be resolved in a specific milestone. Also your query doesn't exclude potential merge into patch branches when patch releases are needed. To me milestone is more about tracking the work required for a release.
